README.md

```diff
@@ -2,40 +2,46 @@
 
 This website is built using [Docusaurus 2](https://docusaurus.io/), a modern static website generator.
 
-### Installation
+## Installation
 
-```
-$ yarn
+```sh
+yarn
```

Comment:
Code fence needs code syntax when appropriate and no `$` unless output also shown for shell